apps/rush/src/RushVersionManager.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import * as fsx from 'fs-extra';
+import * as semver from 'semver';
+
+import { Utilities } from '@microsoft/rush-lib';
+import * as rushLibCli from '@microsoft/rush-lib/lib/start';
+
+import RushWrapper from './RushWrapper';
+
+const RUSH_TRANSITIONAL_VERSION: string = '4.0.0';
+const MAX_INSTALL_ATTEMPTS: number = 5;
```

Comment:
>5; [](start = 37, length = 2)

This seems like a lot of retries for something that (1) is usually manually invoked and (2) is generally a cache hit.

Maybe 3?